app/celer-g4/EventAction.cc

```diff
@@ -75,7 +75,7 @@ void EventAction::EndOfEventAction(G4Event const* event)
     if (GlobalSetup::Instance()->GetWriteSDHits())
     {
         // Write sensitive hits
-        HitRootIO::Instance()->WriteHits(event);
+        RootIO::Instance()->Write(event);
```

Comment:
Still prefer to use the explicit name, `HitRooIO` and `WriteHits` rather than too generic names.

Reply:
I renamed mostly because the purpose expanded. So we were not writing hits exactly, even though we are using `ProcessHits` as the interface to collect the data.